.github/workflows/validations.yaml

```diff
@@ -55,7 +55,7 @@ jobs:
       run: |
         ddev validate jmx-metrics
 
-    - name: Run legacy metadata validation
```

Comment:
Just curious, why the change in wording from legacy to non-legacy?

Reply:
It's a mistake left from copy+pasting the section below. This command just validates metadata (there's no legacy vs non-legacy)